ts/frames.ts

```diff
@@ -1594,6 +1718,12 @@ export const cargoTypes = {
     ['EFreightType::SeedPallet']: 'Seed Pallet',
     ['EFreightType::SteelPipe']: 'Steel Pipes',
     ['EFreightType::StrawBale']: 'Straw Bale',
+    ['EFreightType::Grain']: 'Grain',
+    ['EFreightType::RefinedGold']: 'Refined Gold',
+    ['EFreightType::GoldIngot']: 'Gold Ingot',
+    ['EFreightType::GoldOre']: 'Gold Ore',
+    ['EFreightType::Cattle']: 'Cattle',
+    ['EFreightType::Meat']: 'Meat',
```

Comment:
I'd like to keep this list sorted so it's easier to find a specific entry.

Reply:
meat with farm stuff and so on i guess ?